lib/index_documents.rb

```diff
@@ -0,0 +1,65 @@
+require_relative "../app"
+require 'gds_api/publishing_api_v2'
+
+class IndexDocuments
+  def process(message)
+    links = message.payload["links"].try(:dup)
+
+    if links
+      links.each do |(k, vs)|
```

Comment:
Could you use multi-word names for `k` and `vs` here to make it easier to read the code?  Perhaps `v` is `link_type` and `vs` is `content_ids`?